src/Validator/Constraints/RouteDefaultsValidator.php

```diff
@@ -11,21 +11,25 @@
 
 namespace Symfony\Cmf\Bundle\RoutingBundle\Validator\Constraints;
 
-use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\Controller\ControllerResolverInterface;
 use Symfony\Component\Validator\Constraint;
 use Symfony\Component\Validator\ConstraintValidator;
+use Twig\Loader\LoaderInterface;
 
 class RouteDefaultsValidator extends ConstraintValidator
 {
     private $controllerResolver;
-    private $templating;
 
-    public function __construct(ControllerResolverInterface $controllerResolver, EngineInterface $templating)
+    /**
+     * @var LoaderInterface
+     */
+    private $twig;
+
+    public function __construct(ControllerResolverInterface $controllerResolver, LoaderInterface $twig)
```

Comment:
if we would omit the typehint here, and validate the class in the constructor to be either loader or engine, we could be BC. not sure what the best practice is for that.